### PR TITLE
displayed each item

### PR DESCRIPTION
**Pull request by gracesessions**

### CODE REVIEW

File: db.php [hunk 1]
```diff
@@ -35,8 +35,13 @@ function fetchAll(PDO $dbConnection, string $sql, array $params = null): array
 
 function fetchAllRecordsData(PDO $dbConnection): array
 {
-    $sql = (
-    'SELECT `id`, `name`, `artist`, `year`, `record_label`, `song`, `img_name` FROM `records`;');
+     $sql =
+         'SELECT `records`.`id`, `records`.`name`, `artists`.`artist`, `records`.`year`, `record_labels`.`record_label`, `records`.`song`, `records`.`img_name` 
+         FROM `records`
```
Comment (AndrewScott85) [inline db.php]: indenting of FROM

File: collectionpractice.css [hunk 2]
```diff
@@ -35,17 +37,17 @@ body {
...
     position: absolute;
-    /*align-items: start;*/
-    align-self: start;
+    /*align-items: end;*/
+    align-self: center;
     justify-self: center;
-    margin: 2rem;
+    margin-top: 2.5rem;
+    font-size: 0.9rem;
 }
```
Comment (danieljporter21) [inline collectionpractice.css]: extra lines

File: functions.php [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+            '<div class="recordinfo">' .
+            '<h2>' . $record['name'] . '</h2>' .
+            '<p> By ' . $record['artist'] . '</p>' .
+            '<p> Released in ' . $record['year'] . '</p>' .
+            '<p> by '. $record['record_label'] . '</p>' .
+            '<p>Featuring ' . $record['song'] . '</p>' .
+            '<img src="images/' . $record['img_name'] . '">' .
+            '</div>' .
+            '</div>';
+    } return $info;
```
Comment (danieljporter21) [inline functions.php]: maybe put this return on a seperate line

File: collection.php [hunk 4]
```diff
@@ -12,44 +19,9 @@
 <body>
 <h1>Record Collection</h1>
```
Comment (danieljporter21) [inline collection.php]: indentation

File: collection.php [hunk 5]
```diff
@@ -12,44 +19,9 @@
...
 <section class="collection">
```
Comment (danieljporter21) [inline collection.php]: indentation

File: collection.php [hunk 6]
```diff
@@ -12,44 +19,9 @@
...
-?>
-    </div>
-    <div class="images">
-        <img src='images/londoncalling Small.png'>
-    </div>
-</div>
+    <?php
+    echo displayRecords($records);
+    ?>
 </section>
```
Comment (danieljporter21) [inline collection.php]: indentation

**Review by danieljporter21 — CHANGES_REQUESTED** [review]

**Review by danieljporter21 — APPROVED** [review]

**Pull request merged**